modules/athena/README.md

```diff
@@ -0,0 +1,142 @@
+# Component: `athena`
+
+This component is responsible for provisioning an Amazon Athena workgroup, databases, and related resources.
+
+## Usage
+
+**Stack Level**: Regional
+
+Here are some example snippets for how to use this component:
+
+`stacks/catalog/athena/defaults.yaml` file (base component for all Athena deployments with default settings):
+
+```yaml
+components:
+  terraform:
+    athena/defaults:
+      metadata:
+        type: abstract
+      settings:
+        spacelift:
+          workspace_enabled: true
+      vars:
+        enabled: true
+        tags:
+          Team: sre
+          Service: athena
+        create_s3_bucket: true
+        create_kms_key: true
+        athena_kms_key_deletion_window: 7
+        workgroup_description: ""
+        bytes_scanned_cutoff_per_query: null
+        enforce_workgroup_configuration: true
+        publish_cloudwatch_metrics_enabled: true
+        encryption_option: "SSE_KMS"
+        s3_output_path: ""
+        workgroup_state: "ENABLED"
+        database: []
+
+```
+
+```yaml
+import:
+- catalog/athena/defaults
+
+components:
+  terraform:
+    athena-example: 
```

Comment:
@aknysh I like using `athena/defaults` because it follows the path structure in the catalog. For `athena-example`, it's just the name of the component basically and there is no path.